### PR TITLE
add next epoch validators to header validators; add corresponding pre…

### PR DESCRIPTION
**Pull request by wgr523**
…pare and verify functions

### CODE REVIEW

File: internal/ethapi/api.go [hunk 1]
```diff
@@ -1251,7 +1251,7 @@ func (s *PublicBlockChainAPI) rpcOutputBlock(b *types.Block, inclTx bool, fullTx
 		"timestamp":        (*hexutil.Big)(head.Time),
 		"transactionsRoot": head.TxHash,
 		"receiptsRoot":     head.ReceiptHash,
-		"validators":       hexutil.Bytes(head.Validators),
+		"validators":       hexutil.Bytes(head.Validators.CurrentEpoch),//TODO gerui: ask what should be here!
```
Comment (wgr523) [inline internal/ethapi/api.go]: What should be the new rpc output look like now that I have two lists in "validators"? @wjrjerome  @liam-lai

Comment (liam-lai) [reply internal/ethapi/api.go]: I have no idea,I can try to deploy your branch and see the output

Comment (wgr523) [reply internal/ethapi/api.go]: changed this line comment to `//omit head.Validators.NextEpoch on purpose`

**Review by wjrjerome — APPROVED** [review]

File: internal/ethapi/api.go (hunk 1 above)
Comment (wjrjerome) [reply internal/ethapi/api.go]: i think u going to generate a string hex which we won't be able to recognize. at the end of day, this hexutil function just turn a variable into hex string

Comment (wgr523) [reply internal/ethapi/api.go]: the string is concatenation of addresses. E.g., two address 0x11111111111111111111111111111, 0x22222222222222222222. Then the hex string is 0x1111111111111111111111111111122222222222222222222. Do you think we need to split the string into addresses?

**Pull request merged**